### ISSUE TITLE
[Proposal] Green Reviews Working Group

### ISSUE THREAD

**Issue by nikimanoledaki** [cncf/tag-env-sustainability#116]
#### Description
**This is a proposal to create a new Working Group (WG) under the CNCF Environmental Sustainability TAG (TAG ENV).**

Please see [this document](https://docs.google.com/document/d/18MkLryOSbZpSsgl_ileLbHp_K4xn8TqxMzcHDtNuY_o/edit#heading=h.73pgf6pbinj0) for more info.

### TODO

- [x] TAG Environmental Sustainability lead representative: @leonardpahlke 
- [x] Project leader(s): @nikimanoledaki @guidemetothemoon 
- [ ] Project Members who have contributed to this WG Proposal discussion:
  - Chrystian Blas
  - Max Körbächer
  - Christopher Hanson
  - Antonio Di Turi
  - Eduardo Rivero
  - Alexis Carrel Billiard
  - and many more - thank you! 💚 
- [ ] _Fill in addition TODO items here so the project team and community can see progress!_
- [ ] Scope 
- [ ] Deliverable(s)
- [ ] Project Schedule
- [ ] Slack Channel (as needed)
- [ ] Meeting Time & Day:
- [ ] Meeting Notes (link)
- [ ] Meeting Details (zoom or hangouts link)
- [ ] Retrospective

**Comment by leonardpahlke** [comment]
Thanks for opening this proposal 🎉 I am very excited to see this. 

I think the scope is fine if we limit it to GreenOps. IMO the WG should not be about anything tooling wise, just GreenOps scope. That's mentioned in the proposal, but I just want to highlight it 😄. I think the first task of the working group should be to define GreenOps to make the scope even clearer (which kinds of projects, practices, etc. are part of the WG).

**Comment by guidemetothemoon** [comment]
Very excited for this as well, also to the discussions that can come out of it! 😺
Totally agree with your clarification, Leo, it's good to keep it scoped to what the main goal of the WG is.

**Comment by leonardpahlke** [comment]
@nikimanoledaki, @guidemetothemoon I have added some comments to the google docs file. I like the effort and I think this is an important and interesting topic for the TAG and the wider cloud native community. I think its awesome that you both are taking ownership here ❤️. Before we move on, we need to refine the proposal and the scope a bit more so that we are all on the same page and have a clear idea of where we want this to go. Depending on how the proposal is defined in the end (scope, deliverables, etc.), we might want to start this as a project first and not as a working group. As a project, this effort would be time-limited, i.e., until the deliverables are met, which might be a good fit here. But we can discuss this after we refined the proposal. Sounds good? 

Doc: https://docs.google.com/document/d/18MkLryOSbZpSsgl_ileLbHp_K4xn8TqxMzcHDtNuY_o/edit#
